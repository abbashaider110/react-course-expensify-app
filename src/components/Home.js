import React from 'react';
import ExpenseList from './ExpenseList';
import ExpenseListFilters from './ExpenseListFilters';
import ExpenseSummary from './ExpenseSummary';


const Home = () =>(
    <div>
    <ExpenseSummary/>
    <ExpenseListFilters/>
    <ExpenseList/>

    </div>
    );

export default Home;