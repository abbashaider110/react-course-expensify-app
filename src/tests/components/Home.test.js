import {shallow} from 'enzyme';
import React from 'react';
import Home from '../../components/Home';

test('should render home page', () =>{
    const wrapper = shallow(<Home/>);
    expect(wrapper).toMatchSnapshot();
})