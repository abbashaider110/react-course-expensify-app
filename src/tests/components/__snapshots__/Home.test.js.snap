// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render home page 1`] = `
<div>
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
